web/card-fields/single-card-field.html

```diff
@@ -1,65 +1,260 @@
-<script src="https://localhost.paypal.com:8443/sdk/js?client-id=alc_client1&components=card-fields,buttons"></script>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Add meta tags for mobile and IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>PayPal Single Card Fields | Client Example</title>
 
-<h1>Single Transactional Card Field + Button</h1>
+    <style>
+      * {
+        box-sizing: border-box;
+        font-family: Helvetica;
+      }
 
-<p>
-    Single card field which approves a transaction when the user enters their card number and clicks the button
-</p>
+      .container {
+        width: 445px;
+        margin-bottom: 10px;
+      }
 
-<div id="single-transactional-card-field">
-    <div id="card-field-container" class="container card-field-container"></div>
-    <div id="button-container" class="container button-container"></div>
-</div>
+      .card-field-container {
+        border: 1px solid #909697;
+        border-radius: 3px;
+        line-height: 0px;
+        min-height: 35px;
+        display: none;
+        height: 62px;
+      }
 
-<script>
-    (() => {
-        const container = document.querySelector('#single-transactional-card-field');
+      .valid {
+        border: 1px solid #909697;
+      }
+      .invalid {
+        border: 1px solid red;
+      }
 
-        const cardContainer = container.querySelector('#card-field-container');
-        const buttonContainer = container.querySelector('#button-container');
+      button {
+        width: 100%;
+        height: 48px;
+        background: #000000;
+        border-radius: 25px;
+        color: white;
+        font-size: 18px;
+        border-color: #000000;
+        cursor: pointer;
+      }
+      #errors-container {
+        display: none;
+        margin-bottom: 10px;
+      }
+      .error-icon {
+        color: red;
+      }
+      .loading {
+        transform: translateX(0);
+        animation: 1.5s loading-placeholder ease-in-out infinite;
+        cursor: not-allowed;
+        opacity: 0.5;
+      }
+      @keyframes loading-placeholder {
+        0% {
+          opacity: 0.1;
+        }
+        50% {
+          opacity: 1;
+        }
+        100% {
+          opacity: 0.1;
+        }
+      }
+    </style>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+      integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
 
-        let cardFieldValid = false;
+  <body>
+    <!-- PayPal JS SDK -->
+    <script src="https://www.paypal.com/sdk/js?client-id=alc_client1&components=card-fields,buttons&debug=true&intent=capture"></script>
```

Comment:
```suggestion
    <script src="https://www.paypal.com/sdk/js?client-id=test&components=card-fields,buttons&debug=true&intent=capture"></script>
```

Reply:
The prod JS SDK should use `test` as the client id. I think `alc_client1` only works in stage.